kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -264,9 +264,7 @@ final case class Stream[V, -S](v: Ack < (Emit[Chunk[V]] & S)):
       *   A unit effect that runs the stream without collecting results
       */
     def runDiscard(using tag: Tag[Emit[Chunk[V]]], frame: Frame): Unit < S =
-        ArrowEffect.handle(tag, v.unit)(
-            [C] => (input, cont) => cont(Stop)
-        )
+        runForeach(_ => ())
```

Comment:
I think this is the correct approach:
```suggestion
        ArrowEffect.handle(tag, v.unit)(
            [C] => (input, cont) => cont(Continue())
        )
```